packages/services/src/vckit.service.ts

```diff
@@ -71,6 +71,19 @@ export const issueVC = async ({
     _validateVckitHeaders(headers);
   }
 
+  if (restOfVC?.validUntil) {
+    const validUntil = restOfVC.validUntil;
+    const validFrom = new Date().toISOString();
+
```

Comment:
I think we could allow users to input the validFrom field manually if they don’t want to use the current date and time. This would make the field more flexible.